sycl/test/on-device/basic_tests/use_pinned_host_memory.cpp

```diff
@@ -22,6 +32,27 @@ int main() {
     CGH.single_task<class init_a>([=]() {});
   });
 
+  {
+    int data1[10] = {-1, -1, -1, -1, -1, -1, -1, -1, -1, -1};
+    {
+      buffer<int, 1> a(data1, range<1>(10), {property::buffer::use_host_ptr()});
+      buffer<int, 1> b(
+          range<1>(10),
+          {ext::oneapi::property::buffer::use_pinned_host_memory()});
```

Comment:
Can we have a test which verifies that piMemBufferCreate with PI_MEM_FLAGS_HOST_PTR_ALLOC allocates memory using zeMemAllocHost?